src/jubilation-axis/x-axis.jsx

```diff
@@ -11,26 +12,43 @@ type Props = {
   title?: string,
   position?: number,
   numTicks?: number,
-  tickLines?: boolean,
+  extendTicks?: boolean,
   axisLine?: boolean
 };
 type Context = { JubilationContext: JubilationContext };
 
 export default function XAxis(
-  { min, max, title, position = 0, numTicks = 0, tickLines = false, axisLine = false }: Props,
+  { min, max, title, position = 0, numTicks = 0, extendTicks = false, axisLine = false }: Props,
   { JubilationContext }: Context
   ): React.Element<*> {
   const context = getContext(JubilationContext);
   const computedMin = min || min === 0 ? min : context.xScale.domain()[0];
   const computedMax = max || max === 0 ? max : context.xScale.domain()[1];
-  const ticks = getTicks(computedMin, computedMax, numTicks, 'x', position, context);
+  const ticks = getTicks({
+    min: computedMin,
+    max: computedMax,
+    numTicks,
+    axisType: 'x',
+    position,
+    context,
+  });
   const offset = context.theme.labelStyle.fontSize;
 
   return (
-    <Animation data={{ min: computedMin, max: computedMax, position, offset, ticks }}>
-      {data =>
+    <Animation data={{ min: computedMin, max: computedMax, position, offset, ticks, extendTicks }}>
+      {(data) =>
         <g>
-          {/* tick values */}
+          {
+            data.extendTicks && data.ticks.map((tick) =>
+              <Line
+                x1={tick.x}
+                x2={tick.x}
+                y1={data.min}
```

Comment:
@coreyflynn This brings tick lines to the bottom of the chart, but not particularly close to the label.  What's the desired behavior?

Additionally, right now, ticks are extended or invisible, which I'm pretty sure is not the right behavior, given your comments in the source issue.  Is there a minimum size, and then a "full-width" size?